index.js

```diff
@@ -83,7 +118,14 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+for (let movie of movies){
```

Comment:
Start trying to keep a space between our `()` and the ` { ` to open the block.  It's very nit picky ( and something I work on myself) but it's a good habit to write clean code.